crates/rpc/rpc-builder/src/lib.rs

```diff
@@ -254,11 +250,11 @@ where
     EvmConfig: ConfigureEvm,
 {
     let module_config = module_config.into();
-    let server_config = server_config.into();
-    RpcModuleBuilder::new(provider, pool, network, executor, events, evm_config)
-        .build(module_config)
-        .start_server(server_config)
-        .await
+    let mut server_config = server_config.into();
+    let modules = RpcModuleBuilder::new(provider, pool, network, executor, events, evm_config)
+        .build(module_config);
+    let handle: RpcServerHandle = server_config.start_ws_http(&modules).await?;
```

Comment:
this two step shouldn't be required, this should remain one single chain build + start

Reply:
is the updated code what you had in mind?